### PR TITLE
Docs: direct Github links to master

### LINKED ISSUE #2308: Direct all GitHub links in documentation to `master` branch
### Edit existing content 📝

**Which page(s) do you want us to edit?**
Multiple

**What do you think should be changed?**
Any links in the documentation that redirect to our GitHub repo should go to the `master` branch, not the `develop` branch. Almost always, `develop` has lots of changes as compared to `master`. This is especially important for everything in the contributing section.

Related: #2132

### PR DESCRIPTION
**Pull request by Akhil-Sharma30**
# What does this PR do?

Fixes #2308.

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Comment by Akhil-Sharma30** [comment]
Please let me know if there are any other links to change.

File: docs/content/usage/ransomware-simulation.md [hunk 1]
```diff
@@ -186,4 +186,4 @@ the victim machine in order to replicate this behavior.
 The README.txt file informs the user that a ransomware simulation has taken
 place and that they should contact their administrator. The contents of the
 file can be found
-[here](https://github.com/guardicore/monkey/blob/develop/monkey/agent_plugins/payloads/ransomware/src/ransomware_readme.txt).
+[here](https://github.com/guardicore/monkey/blob/master/monkey/infection_monkey/payload/ransomware/ransomware_readme.txt).
```
Comment (mssalvatore) [inline docs/content/usage/ransomware-simulation.md:189]: This URL won't exist after we cut the next release. Just change `develop` to `master` for this.

**Review by mssalvatore — CHANGES_REQUESTED** [review]

**Comment by mssalvatore** [comment]
Missing `setup-developmient-environment.py`

**Comment by Akhil-Sharma30** [comment]
I wanted to propose a feature for the repository.

Should I add a `link checker` to the project? It would help in the validation of the link present in the repo

**Pull request merged**